src/bridge.js

```diff
@@ -34,7 +34,7 @@ export default class Bridge extends EventEmitter {
       this._wall.send(event, payload, transferable);
       this._time = Date.now();
     } else {
-      this._messageQueue.push(event, payload, transferable);
+      this._messageQueue.push({ event, payload, transferable });
```

Comment:
Can you explain why this change is actually important? Looks to me like all you're doing is wrapping args  in an object and unwrapping them below, but why?